arches/app/etl_modules/branch_excel_importer.py

```diff
@@ -146,20 +146,24 @@ def process_worksheet(self, worksheet, cursor, node_lookup, nodegroup_lookup):
         )
         return {"name": worksheet.title, "rows": row_count}
 
+    def validate_uploaded_file(self, file):
+        workbook = load_workbook(filename=default_storage.open(file))
+        graphid = None
+        try:
+            graphid = workbook.get_sheet_by_name("metadata")["B1"].value
+        except: # KeyError
```

Comment:
Could you change this to except the KeyError?
`except KeyError:`

Reply:
This is intentionally not catching specific error, since any unexpected error makes the status look like it hangs. (either fail nor succeed)

Reply:
I think for the time being we should handle for the specific error. This gives us a chance to learn what other types of errors can occur so that we can figure out how to prevent them and handle for them properly.